### PR TITLE
[9.0][FIX] password_security: noupdate rule

### PR DESCRIPTION
**Pull request by chienandalu**
Record rules should be noupdate

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Change module version

File: password_security/__openerp__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     'name': 'Password Security',
     "summary": "Allow admin to set password security requirements.",
-    'version': '9.0.1.0.3',
+    'version': '9.0.1.0.4',
```
Comment (pedrobaeza) [inline password_security/__openerp__.py]: As this requires module update, it should be 9.0.1.1.0. You can squash already the commits as the history is very simple.

**Comment by chienandalu** [comment]
@pedrobaeza Changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by lasley — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Fast-tracking this.

**Pull request merged**